### DISCUSSION TITLE
https://uvicorn.org/ should redirect to https://www.uvicorn.org/

### DISCUSSION THREAD

**Discussion by SamuelMarks** [Kludex/uvicorn#2318] [Q&A]
### Initial Checks

- [X] I confirm this was discussed, and the maintainers suggest I open an issue.
- [X] I'm aware that if I created this issue without a discussion, it may be closed without a response.

### Discussion Link

```Text
N/A
```

### Description

N/A

### Example Code

```Python
N/A
```

### Python, Uvicorn & OS Version

```Text
N/A
```

> [!IMPORTANT]
> - We're using [Polar.sh](https://polar.sh/encode) so you can upvote and help fund this issue.
> - We receive the funding once the issue is completed & confirmed by you.
> - Thank you in advance for helping prioritize & fund our backlog.

<a href="https://polar.sh/encode/uvicorn/issues/2296">
<picture>
  <source media="(prefers-color-scheme: dark)" srcset="https://polar.sh/api/github/encode/uvicorn/issues/2296/pledge.svg?darkmode=1">
  <img alt="Fund with Polar" src="https://polar.sh/api/github/encode/uvicorn/issues/2296/pledge.svg">
</picture>
</a>

**Comment by Kludex** [comment]
@tomchristie Can you do something here?
